front/app/components/IdeaButton/index.tsx

```diff
@@ -177,13 +176,20 @@ const IdeaButton = memo<Props & WrappedComponentProps>(
       if (!isNilOrError(project)) {
         trackEventByName(tracks.redirectedToIdeaFrom);
 
+        const isUserModerator =
+          !isNilOrError(authUser) &&
+          canModerateProject(projectId, { data: authUser });
+
+        const parameters =
+          phaseId && isUserModerator ? `&phase_id=${phaseId}` : '';
+
```

Comment:
We only ever want to add a phase ID to the URL when the user is a project moderator/admin.